src/scripts/main.js

```diff
@@ -1,3 +1,92 @@
 'use strict';
 
 // write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const table = document.querySelector('.field tbody');
+
+const minCell = 2;
+const maxCell = 10;
+
+function addRow() {
+  const rows = table.querySelectorAll('tr');
+  const lastRow = table.querySelector('tr:last-child');
+  const newRow = lastRow.cloneNode(true);
+
+  if (rows.length < maxCell) {
+    removeRow.disabled = false;
+    table.appendChild(newRow);
+  }
+
+  if (rows.length + 1 === maxCell) {
+    appendRow.disabled = true;
+  }
+}
+
+function addCol() {
+  const rows = table.querySelectorAll('tr');
+  const countCell = rows[0].querySelectorAll('td');
```

Comment:
This line assumes that there is at least one row in the table. If the table can be empty, you should add a check to ensure that `rows[0]` exists before trying to access its cells.